src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Mapping/Mapping.ts

```diff
@@ -150,6 +149,16 @@ export var mapInput = (
                 scope.mapClicked = false;
                 scope.text = "TR__MAP_EXPLAIN_CLICK";
             };
+
+            scope.$watch((<any>map), function(){
```

Comment:
AFAIK the first argument need to be either a string or a function. Why do you watch map by the way? I guess what we really want to watch is the element width, right?